### ISSUE TITLE
Why do unconnected vertices disappear?

### ISSUE THREAD

**Issue by jhannah-mm** [ronsavage/GraphViz2#10]
```
use Graph::Directed;
use GraphViz2;
my $g = Graph::Directed->new;
$g->add_edges(1, 2);
$g->add_vertex(3);
GraphViz2->from_graph($g)->run(format => 'png', output_file => 'demo3.png');
```
![demo3](https://user-images.githubusercontent.com/40041975/108535198-ad22b980-72a0-11eb-81f2-b99af20580b7.png)

**Comment by mohawk2** [comment]
I'd made the DOT output be leaner by only doing `add_node` when there were attributes, having missed the case of isolated vertices. About to release fixed version. Thanks!

**Comment by jhannah-mm** [comment]
Confirmed fixed in GraphViz2-2.66. THANKS!
